fix layout() source folder description (#2867)
conan-io/docs: reference/conanfile/methods.rst

- **self.folders.source** (Defaulted to ""): Specifies a subfolder where the sources are. The ``self.source_folder`` attribute

>   inside the ``source(self)`` and ``build(self)`` methods will be set with this subfolder. The *current working directory*
>   in the ``source(self)`` method will include this subfolder. As well, the `export_sources`, `exports` and `scm` sources
>   will be copied to the root source directory.

Reviewer: ```suggestion
  inside the ``source(self)`` and ``build(self)`` methods will be set with this subfolder. The *current working directory*
  in the ``source(self)`` method will include this subfolder. The `export_sources`, `exports` and `scm` sources
  will also be copied to the root source directory.
```
